funnel/signals.py

```diff
@@ -45,6 +45,7 @@
 org_data_changed = app_signals.signal('org-data-changed')
 team_data_changed = app_signals.signal('team-data-changed')
 session_revoked = app_signals.signal('session-revoked')
+project_email_dispatched = app_signals.signal('project_email_dispatched')
```

Comment:
Incorrect name. No email has been dispatched at the time this signal is raised. This signal indicates that a user has registered for a project, so name it appropriately.